lib/TopStoriesFeed.jsm

```diff
@@ -113,13 +113,18 @@ this.TopStoriesFeed = class TopStoriesFeed {
     this.store.dispatch(shouldBroadcast ? ac.BroadcastToContent(action) : ac.AlsoToPreloaded(action));
   }
 
-  doContentUpdate(shouldBroadcast) {
+  doContentUpdate({stories, topics}, shouldBroadcast) {
     let updateProps = {};
-    if (this.stories) {
-      updateProps.rows = this.stories;
+    if (stories) {
+      updateProps.rows = stories;
+    } else {
+      const {Sections} = this.store.getState();
```

Comment:
Add a comment on why we need to get the previous `rows` especially because we don't need to for `topics` below.